### DISCUSSION TITLE
UNMAPPED DOCKER PATH MAPPING FILE LOCATION?

### DISCUSSION THREAD

**Discussion by TheQwenton** [nextcloud/all-in-one#2905] [Questions] [answered]
I have a weird question.. Nextcloud AIO instance installation requires this pathmapping to not be changed. Where exactly is this being pointed to? There is a config file being generated there that I need to edit. Specifically asking where this file would be located within the unraid file structure. I have searched everywhere in krusader and cannot find it. Please advise!

![image](https://github.com/nextcloud/all-in-one/assets/119769882/ccfb31f4-c656-4b22-9853-4746016321d3)

/mnt/docker-aio-config <---> nextcloud_aio_mastercontainer

**Comment by elephantastyczny** [comment, marked as answer]
https://github.com/nextcloud/all-in-one/pull/2406/files/378c884442ee8ce10347cf4f1374acad1fd26ac2#diff-92cbbe5ed1f9f8e44fcc68c0c451428f8c5d30b934f605a338cd7d0f5bf4d36e

 <summary>Click here to view the current AIO config and documentation links</summary><br />
    {% if domain != '' %}
        Nextclouds config.php file is stored in the nextcloud_aio_nextcloud Docker volume and can by edited by following <b><a href="https://github.com/nextcloud/all-in-one#how-to-edit-nextclouds-configphp-file-with-a-texteditor">this documentation</a></b><br><br>
        You can run Nextcloud's usual occ commands by following <b><a href="https://github.com/nextcloud/all-in-one#how-to-edit-nextclouds-configphp-file-with-a-texteditor">this documentation</a></b><br><br>

**Reply by TheQwenton** [reply]
Thank you for this. Was able to find the file (config.php) within unraid using this command `sudo docker run -it --rm --volume nextcloud_aio_nextcloud:/var/www/html:rw alpine sh -c "apk add --no-cache nano && nano /var/www/html/config/config.php"`
